infrastructure/main.tf

```diff
@@ -15,5 +19,13 @@ module "bulk-scan-ccd-event-handler-sample-app" {
   common_tags         = "${var.common_tags}"
 
   app_settings = {
+    S2S_URL           = "${local.s2s_url}"
+    S2S_NAME          = "${var.s2s_name}"
+    S2S_SECRET        = "${data.azurerm_key_vault_secret.s2s_key.value}"
```

Comment:
will dummy app sign in to s2s?

Reply:
How else will it create new applications in CCD? The library also requires a token supplier.